roles/ceph-fetch-keys/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: find ceph keys
+  shell: ls -1 /etc/ceph/*.keyring
+  changed_when: false
+  register: ceph_keys
+
+- name: set keys permissions
+  file:
+    path: "{{ item }}"
+    mode: 0600
+    owner: root
```

Comment:
Getting this error is normal while running your command like (without other argument), it implies you use `--name client.admin` and then it looks for `/etc/ceph/client.admin.keyring`.

OSD nodes shouldn't have the admin key so you're all good :)

I think it's fine keeping root as user for the keys in `/etc/ceph` as it only contains the admin one. So nop we are good :)